matrix-sdk-android/src/main/java/org/matrix/android/sdk/api/session/events/model/Event.kt

```diff
@@ -447,3 +448,11 @@ fun Event.supportsNotification() =
 
 fun Event.isContentReportable() =
         this.getClearType() in EventType.MESSAGE + EventType.STATE_ROOM_BEACON_INFO.values
+
+fun Event.getIdsOfPinnedEvents(): MutableList<String>? {
```

Comment:
This should return a `List` instead of a `MutableList`. There are other similar remarks in your PR, I will just comment the others with `List`.